### PR TITLE
Fix/bandwidth apac

### PR DESCRIPTION
**Pull request by marie-j**
ref: DTRSD-4549

### CODE REVIEW

**Review by cbourgois — DISMISSED** [review]

**Review by FredericEspiau — DISMISSED** [review]

File: client/app/dedicated/server/consumption/consumption.html [hunk 1]
```diff
@@ -34,22 +34,18 @@ <h4 class="oui-header_4 oui-tile__title">
                 <dl class="oui-tile__definition">
                     <dt class="oui-tile__term"
                         data-ng-bind="'server_consumption_current_period_label' | translate:{ t0: ctrl.traffic.data.usage.outputQuotaSize.text }"></dt>
-                    <dd class="oui-tile__description"
+                        <dd class="oui-tile__description"
```
Comment (antleblanc) [inline client/app/dedicated/server/consumption/consumption.html]: Weird indentation.

**Comment by ovh-cds** [comment]
CDS Report build-manager#3263.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✔ 
  * Build manager US ✘  
  * Tests ✔

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_3#3263.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_2#3263.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci#3263.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by marie-j** [comment]
Closing this as it is handled by https://github.com/ovh/manager/pull/1638

**Pull request closed without merging**